mne/_fiff/meas_info.py

```diff
@@ -2206,7 +2206,10 @@ def read_meas_info(fid, tree, clean_bads=False, verbose=None):
             description = tag.data
         elif kind == FIFF.FIFF_PROJ_ID:
             tag = read_tag(fid, pos)
-            proj_id = tag.data
+            if isinstance(tag.data, np.ndarray):
+                proj_id = int(tag.data.item())
+            else:
+                proj_id = int(tag.data)
```

Comment:
I don't think you should need this conditional, it should always be an `ndarray` I think
```suggestion
            proj_id = int(tag.data.item())
```